jme3-testdata/src/main/resources/Scenes/LightProbes/license.txt

```diff
@@ -0,0 +1,16 @@
+Licensing history for files in "src/main/resources/Textures/Sky/quarry_03"
```

Comment:
file path looks incorrect

Reply:
I've updated it to match the proper location